lib/board.rb

```diff
@@ -0,0 +1,51 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells
+   
+    def initialize
+      @cells=[]
```

Comment:
`reset!` will take care of setting `@cells`, no need to also set it equal to an empty array.